### ISSUE TITLE
Does helm operator implement controller

### ISSUE THREAD

**Issue by bitscuit** [operator-framework/operator-sdk#4711]
## Type of question

General operator-related help

## Question
I have upgraded to operator-sdk 1.5.0 from 1.3.0, and I read there was a breaking change in the release notes

> Breaking change: PROJECT config version 3-alpha has been upgraded to version 3. (#4613)

After running `operator-sdk alpha config-3alpha-to-3`, my PROJECT file has this TODO item
```
 # TODO(user): Uncomment the below line if this resource implements a controller, else delete it.
```

My operator is a helm based one, so would it be considered as implementing a controller?

#### Environment

**Operator type:**

/language helm

**Kubernetes cluster type:**

`$ operator-sdk version`

operator-sdk version: "v1.5.0", commit: "98f30d59ade2d911a7a8c76f0169a7de0dec37a0", kubernetes version: "1.19.4", go version: "go1.15.5", GOOS: "linux", GOARCH: "amd64"

`$ go version` (if language is Go)

`$ kubectl version`

**Comment by bitscuit** [comment]
Just looking for a "yes/no" answer to this, so I can test if my operator upgrades run smoothly

**Comment by camilamacedo86** [comment]
Hi @bitscuit, 

You have no controllers. The issue is in the migration tool. You can see in the [testdata/helm/memcached-operator/PROJECT](https://github.com/operator-framework/operator-sdk/blob/v1.5.0/testdata/helm/memcached-operator/PROJECT) an example how it should be.

**Comment by jmrodri** [comment]
The bug here is that the `alpha config-3alpha-to-3` puts a TODO that is confusing to helm and ansible developers since there is no action for them. The fix for this will be to remove this TODO comment from helm and ansible based configs to avoid the confusion in the future.

**Comment by bitscuit** [comment]
Thank you for the explanations!

**Comment by estroz** [comment]
Closed by #4742
